terms.html

```diff
@@ -80,7 +81,7 @@
 </ul>
 <p><span class="section-number">26</span> The announcement will give a date on which the new terms will come into effect. Except in situations forced on us by unexpected changes in the law, we will always give at least one month's notice of any change.</p>
 <p><span class="section-number">27</span> If you are using our free pricing plan, your continued use of our services indicates your acceptance of the revised terms.</p>
-<p><span class="section-number">28</span> If you are using any other pricing plan and we have announced changes to our terms and conditions, you may terminate your contract with us by giving us notice in writing within 21 days of the date on which the announcement was made.  Under these circumstances, your right of access to <em>the platform</em> will terminate on the date the new terms and conditions come into force. You will not be entitled to a refund in respect of any money already paid to us.</p>
+<p><span class="section-number">28</span> If you are using any other pricing plan and we have announced changes to our terms and conditions, you may terminate your contract with us by giving us notice in writing within 21 days of the date on which the announcement was made.  Under these circumstances, your right of access to PDF Tables will terminate on the date the new terms and conditions come into force. You will not be entitled to a refund in respect of any money already paid to us.</p>
 <h2>Termination</h2>
 <p><span class="section-number">29</span> If you are using this service under the terms of our free pricing plan, you may terminate your use of the service at any time by closing your account from within the account options page.</p>
```

Comment:
We don't have such a button at the moment, but we should have in the long run. Does an omission like that on our site cause problems?